### PR TITLE
Add input docs

### PR DESCRIPTION
**Pull request by streeve**
Tried to put in a reasonable draft, but I probably got some things wrong

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+- `total_output_steps`: Frequency to output grid information
+  - units: unitless
+- `total_monitor_steps`: Frequency to output timing information
+  - units: unitless
+
+## Spatial parameters (`space`)
+
+- `initial_temperature`: Initial temperature at all grid points
+  - units: `K`
+- `cell_size`: Grid cell size (per dimension)
```
Comment (MattRolchigo) [inline examples/README.md:27]: (per dimension) seems confusing here, a user may think this means that we support cell sizes that are non-cubic

File: examples/README.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  - units: `J/kg`
+- `solidus`: Solidus temperature
+  - units: `K`
+- `liquidus`: Liquidus temperature
+  - units: `K`
+
+## Laser source parameters (`source`)
+- `absorption`: Laser absorption
+  - units: unitless
+- `two_sigma`: Beam size
```
Comment (MattRolchigo) [inline examples/README.md:55]: John can correct me if I'm wrong here but I think two_sigma is half of the beam width, not the beam width itself

File: examples/README.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+  - units: `m`
+- `scan_path_file`: File containing laser path information
+
+
+
+## Output sampling (`sampling`)
+This section is optional
+
+- `type`: Type of sampling
+  - options: `solidification_data` (outputs sampled solidification data with spatial position x, y, z; melting time tm; solidification time ts; solidification rate R; and (optionally) temperature gradients Gx, Gy, Gz)
```
Comment (MattRolchigo) [inline examples/README.md:65]: I'd call ts "solidification start time" (or the time at which the location goes below the liquidus temperature)

**Review by MattRolchigo — APPROVED** [review]

**Comment by streeve** [comment]
@colemanjs can you make sure I didn't say anything too dumb?

File: examples/README.md (hunk 2 above)
Comment (streeve) [reply examples/README.md]: @colemanjs is this correct?

Comment (colemanjs) [reply examples/README.md]: it is two sigma. The ISO definition for beam size is diameter at four sigma (D4σ). So 'two_sigma' is the beam radius.

**Comment by colemanjs** [comment]
Everything looks

**Review by colemanjs — APPROVED** [review]
Everything looks good

**Pull request merged**